### PR TITLE
Ae 92 implement recruiter sign up

### PR DESCRIPTION
**Pull request by otimebhor**

### CODE REVIEW

**Review by Mbaoma — APPROVED** [review]

File: src/recruiter/recruiter.controller.spec.ts
Comment (Tes-program) [inline src/recruiter/recruiter.controller.spec.ts:1]: Where are your tests?

File: src/recruiter/recruiter.controller.ts
Comment (Tes-program) [inline src/recruiter/recruiter.controller.ts:1]: Where are the controllers?

File: src/recruiter/recruiter.service.spec.ts
Comment (Tes-program) [inline src/recruiter/recruiter.service.spec.ts:1]: What happened to the tests?

**Review by Tes-program — CHANGES_REQUESTED** [review]

File: src/auth/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    // Remove the onboarding key from redis
+    await this.redis.hdel('onboarding', keyId);
+
+    return {
+      message: 'User created successfully',
+    };
+  }
+
+  async addRecruiter(userId: string, createUserDto: CreateAccountDto) {
+    const admin = await this.userRepository.findOne({
```
Comment (Tes-program) [inline src/auth/auth.service.ts:214]: There is already a auth.repository.ts for database methods. I thought we were already clear on using Repository Pattern na?

**Review by Tes-program — CHANGES_REQUESTED** [review]

**Review by Tes-program — APPROVED** [review]

**Review by Mbaoma — APPROVED** [review]

**Pull request closed without merging**